sample/src/App.js

```diff
@@ -38,20 +38,29 @@ const Card = ({ title, children }) => {
 // fetching them now
 const preloadSDKs = firebaseApp => {
   return Promise.all([
-    preloadFirestore(firebaseApp, firestore => {
-      return firestore().enablePersistence();
+    preloadFirestore({
+      firebaseApp,
+      setup(firestore) {
```

Comment:
any opinion on 

```js
setup(firestore) {
```

vs

```js
setup: firestore => {
```

for all of these?

The second is a bit more clear to me, but that could just be me

Reply:
I prefer the named function. AFIK, it will show up in the stack trace and the arrow function will not.